docs/kss-assets/kss-markup.js

```diff
@@ -24,11 +31,19 @@
 				el.removeAttribute('open');
 			} else {
 				el.setAttribute('open', '');
+				this.highlightMarkup(el);
 			}
 		}
 		// Toggle the markup mode.
 		body.classList.toggle(this.bodyClass);
 	};
+
+	KssMarkup.prototype.highlightMarkup = function(element) {
+		if (!element.classList.contains('prism-highlighted')) {
+			window.Prism.highlightAllUnder(element);
+			element.currentTarget.classList.add('prism-highlighted');
```

Comment:
Isn't the `currentTarget` superfluous here, or do I have forgotten something?

Reply:
good catch! I refactored the highlighting and accidentally left that in while adjusting. Fixed.